mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/platform.kt

```diff
@@ -9,3 +9,5 @@ internal expect inline fun AtomicBoolean.set(value: Boolean)
 internal expect inline fun AtomicBoolean.compareAndSet(expect: Boolean, update: Boolean): Boolean
 
 internal expect inline fun atomicBooleanOf(initialValue: Boolean): AtomicBoolean
+
+internal expect inline fun nanoTime(): Long
```

Comment:
Alternatively, it was possible to use [TimeMark](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-time-mark/) from Kotlin, which under the hood will use the most accurate time available in the system. There would be a little more code in commonMain, but without the code in the platform sources.

Reply:
By capturing an instance and then calling `elapsedNow().inWholeNanoseconds`? My problem with that is it has to make this low-level call to get the current time and then do the substraction, pack the result into a double, and then we unpack the double. I'd rather just make the call directly.

Now once kotlinx.datetime's `Instant` moves to the stdlib... maybe we can switch to that and eliminate all this platform stuff.